app/ui-utils/client/lib/reactionListContent.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Box, Tag, Divider } from '@rocket.chat/fuselage';
+
+import { useTranslation } from '../../../../client/contexts/TranslationContext';
+import Emoji from '../../../../client/components/basic/Emoji';
+
+export function Reactions(props) {
+	const styleDivider = {
+		marginTop: '8px',
+		marginBottom: '16px',
```

Comment:
Prefer the [logical property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties):

```suggestion
		marginBlockStart: '8px',
		marginBlockEnd: '16px',
```